src/playlist.tsx

```diff
@@ -109,6 +92,44 @@ export class Playlist extends KalturaPlayer.core.BasePlugin {
     }
   }
 
+  private _addSidePanel(){
+    if (!this._loaded) {return}
+
+    const pluginMode = [SidePanelPositions.RIGHT, SidePanelPositions.LEFT].includes(this.config.position)
+    || [PLAYER_SIZE?.SMALL,PLAYER_SIZE?.EXTRA_SMALL, PLAYER_SIZE?.TINY].includes(this._player.ui.store.getState().shell.playerSize)
+        ? PluginPositions.VERTICAL
+        : PluginPositions.HORIZONTAL;
+
+    if (this.pluginMode === pluginMode) {
+      return
```

Comment:
your linter  / prettier doesnt add ; automatically ?
we do use ; on every line so please them to be consistent

Reply:
fixed